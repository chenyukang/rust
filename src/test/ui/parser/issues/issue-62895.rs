fn main() {}

fn v() -> isize { //~ ERROR mismatched types
mod _ { //~ ERROR expected identifier
pub fn g() -> isizee { //~ ERROR cannot find type `isizee` in this scope
mod _ { //~ ERROR expected identifier
pub    g() -> is //~ ERROR missing `fn` for function definition
(), w20); //~ ERROR mismatched closing delimiter
}
(), w20); //~ ERROR mismatched closing delimiter
//~^ ERROR expected item
}
