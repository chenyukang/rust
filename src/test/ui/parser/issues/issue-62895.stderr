error: unexpected closing delimiter: `)`
  --> $DIR/issue-62895.rs:8:8
   |
LL | fn main() {}
   |           -- ...matches this closing brace
   |           |
   |           this opening brace...
...
LL | (), w20);
   |        ^ unexpected closing delimiter

error: unexpected closing delimiter: `)`
  --> $DIR/issue-62895.rs:10:8
   |
LL | mod _ {
   |       - this opening brace...
...
LL | }
   | - ...matches this closing brace
LL | (), w20);
   |        ^ unexpected closing delimiter

error: this file contains an unclosed delimiter
  --> $DIR/issue-62895.rs:11:3
   |
LL | fn v() -> isize {
   |                 - unclosed delimiter
LL | mod _ {
   |       - unclosed delimiter
...
LL | }
   |   ^

error: this file contains an unclosed delimiter
  --> $DIR/issue-62895.rs:11:3
   |
LL | fn v() -> isize {
   |                 - unclosed delimiter
LL | mod _ {
   |       - unclosed delimiter
...
LL | }
   |   ^

error: expected identifier, found reserved identifier `_`
  --> $DIR/issue-62895.rs:4:5
   |
LL | mod _ {
   |     ^ expected identifier, found reserved identifier

error: expected identifier, found reserved identifier `_`
  --> $DIR/issue-62895.rs:6:5
   |
LL | mod _ {
   |     ^ expected identifier, found reserved identifier

error: missing `fn` for function definition
  --> $DIR/issue-62895.rs:7:4
   |
LL | pub    g() -> is
   |    ^^^^
   |
help: add `fn` here to parse `g` as a public function
   |
LL | pub fn g() -> is
   |     ++

error[E0412]: cannot find type `isizee` in this scope
  --> $DIR/issue-62895.rs:5:15
   |
LL | pub fn g() -> isizee {
   |               ^^^^^^ help: a builtin type with a similar name exists: `isize`

error[E0308]: mismatched types
  --> $DIR/issue-62895.rs:3:11
   |
LL | fn v() -> isize {
   |    -      ^^^^^ expected `isize`, found `()`
   |    |
   |    implicitly returns `()` as its body has no tail or `return` expression

error: aborting due to 9 previous errors

Some errors have detailed explanations: E0308, E0412.
For more information about an error, try `rustc --explain E0308`.
